examples/play_buffer.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python3
+
+import argparse
+import ctypes as ct
```

Comment:
This saves 4 bytes, but hampers readability. I think it would be nice to keep a plain `import ctypes`